src/Console/Command/HelpCommand.php

```diff
@@ -376,6 +380,21 @@ protected function initialize(InputInterface $input, OutputInterface $output)
     }
 
     /**
+     * @return string|null
+     */
+    private static function getChangeLogFile()
+    {
+        $changelogFile = __DIR__.'/../../../CHANGELOG.md';
+        if (!is_file($changelogFile)) {
+            return null;
+        }
+
+        $changelogFile = realpath($changelogFile);
```

Comment:
That is what I want to try out here as the file should always exists (both phar and non phar). For non-phar it should always be readable, for phar it is not (that is reported). Using `realpath` should result OK for non-phar and `false` for phar so the fallback would be used. This way we won't hide a failed readable check for non-phar.
Lets see what happens ;)